crate/src/plugin.rs

```diff
@@ -2,35 +2,39 @@ use bevy_app::prelude::*;
 use bevy_ecs::prelude::*;
 
 use crate::{
-    IosGCAchievementProgressResponse, IosGCAchievementsResetResponse, IosGCAuthResult,
+    request, IosGCAchievementProgressResponse, IosGCAchievementsResetResponse, IosGCAuthResult,
     IosGCDeleteSaveGameResponse, IosGCFetchItemsForSignatureVerificationResponse,
-    IosGCLoadGamesResponse, IosGCPlayer, IosGCSaveGamesResponse, IosGCSavedGameResponse,
-    IosGCScoreSubmitResponse,
+    IosGCLoadGamesResponse, IosGCPlayer, IosGCResolvedConflictsResponse, IosGCSaveGames,
+    IosGCSaveGamesResponse, IosGCSavedGameResponse, IosGCScoreSubmitResponse,
 };
 
 /// All events for communication from native iOS (Swift) side to Rust/Bevy
 #[derive(Event, Clone, Debug)]
 pub enum IosGamecenterEvents {
     /// Triggered by calls to [`init`][crate::init] or implicit when registering [`IosGamecenterPlugin`] via `IosGamecenterPlugin::new(true)`
-    Authentication(IosGCAuthResult),
+    Authentication((i64, IosGCAuthResult)),
     /// Triggered by calls to [`save_game`][crate::save_game]
-    SavedGame(IosGCSavedGameResponse),
+    SavedGame((i64, IosGCSavedGameResponse)),
     /// Triggered by calls to [`fetch_save_games`][crate::fetch_save_games]
-    SaveGames(IosGCSaveGamesResponse),
+    SaveGames((i64, IosGCSaveGamesResponse)),
     /// Triggered by calls to [`load_game`][crate::load_game]
-    LoadGame(IosGCLoadGamesResponse),
+    LoadGame((i64, IosGCLoadGamesResponse)),
     /// Triggered by calls to [`delete_savegame`][crate::delete_savegame]
-    DeletedSaveGame(IosGCDeleteSaveGameResponse),
+    DeletedSaveGame((i64, IosGCDeleteSaveGameResponse)),
     /// Triggered by calls to [`request_player`][crate::request_player]
-    Player(IosGCPlayer),
+    Player((i64, IosGCPlayer)),
     /// Triggered by calls to [`achievement_progress`][crate::achievement_progress]
-    AchievementProgress(IosGCAchievementProgressResponse),
+    AchievementProgress((i64, IosGCAchievementProgressResponse)),
     /// Triggered by calls to [`achievements_reset`][crate::achievements_reset]
-    AchievementsReset(IosGCAchievementsResetResponse),
+    AchievementsReset((i64, IosGCAchievementsResetResponse)),
     /// Triggered by calls to [`leaderboards_score`][crate::leaderboards_score]
-    LeaderboardScoreSubmitted(IosGCScoreSubmitResponse),
+    LeaderboardScoreSubmitted((i64, IosGCScoreSubmitResponse)),
     /// Triggered by calls to [`fetch_signature`][crate::fetch_signature]
-    ItemsForSignatureVerification(IosGCFetchItemsForSignatureVerificationResponse),
+    ItemsForSignatureVerification((i64, IosGCFetchItemsForSignatureVerificationResponse)),
+    /// Triggered by calls to [`fetch_save_games`][crate::fetch_save_games] or [`save_game`][crate::save_game]
```

Comment:
I'm still on `fe5ac99`, and I'm receiving `ConflictingSaveGames` 5 seconds after I call `save_game` and it isn't mutually exclusive with `SavedGame`. Maybe that's expected, but it seemed weird, so I thought I'd let you know

Reply:
> receiving ConflictingSaveGames 5 seconds after I call

there is really no waiting involved in the code triggered on native side to report to us. but i observe a whole lot of slow behaviour with gamecenter in debug simulator builds. sometimes fetchsavegames just never returns for me

>  it isn't mutually exclusive with `SavedGame`

what does that mean?